test/functional/testUtil/test.js

```diff
@@ -11,7 +11,8 @@ function readFileSync(...paths) {
 }
 
 const sitePath = process.argv[2];
-const expectedDirectory = path.join(sitePath, 'expected');
+const expectedPath = process.argv[3];
+const expectedDirectory = expectedPath === undefined ? path.join(sitePath, 'expected') : expectedPath;
```

Comment:
This seems rather unnecessary. `test.js` has no business in knowing that `test_site_convert` exist. Instead, `sitePath` should be `test_site_convert_expected`, so that this addition is not needed.

Please also rename the directories accordingly:

* `test_site_convert/` -> `test_site_convert/non_markbind_site/` 
* `test_site_convert_expected/` -> `test_site_convert/`

Reply:
After renaming the directories as suggested, I'm not sure I understand...if the `sitePath` is `test_site_convert`, won't the conversion be run on this folder instead of the `non_markbind_site`?

Reply:
Sorry, I forgot that the `init` was in place. Is it not possible to modify `test.sh/bat` to run init on `test_site_convert/non_markbind_site/`, then copy the output files to `test_site_convert/`?

My goal here is to make the test as self-contained as possible, so that (1) there would not be two different separate folders for the same test case, and that (2) we can reuse the original logic of comparing with the `expected/` folder. This is to make the maintenance easier so that people don't get confused about a different test folder structure.

Reply:
Updated to follow logic you've mentioned, please take a look!